### DISCUSSION TITLE
Only show enabled consoles on menu

### DISCUSSION THREAD

**Discussion by PsychoFox11** [sanni/cartreader#815] [Ideas & Suggestions]
I was surprised I didn't see this listed, so apologies if it's been brought up. I just don't understand why consoles that are disabled in the cfg file are still listed on the menu. All they do is show a message telling you to enable them. If you want to list them all just so we don't forget what we can do, put the disabled ones at the end of the list. I'm using the defaults for version 5, plus enabled systems for 5 adapters. Scrolling would be a lot more concise if I didn't have to go through disabled systems. Not a huge pain by any means, but since you gotta pick the console on every power on (which I do between every dump as I've had issues otherwise), and many games take multiple dumps to get right or troubleshoot, it can add up. Also requesting as it seems like it would be a simple fix, though working in software, I know that what looks simple isn't always. It does seem like the menu could just skip anything flagged disabled though.

**Comment by sanni** [comment]
I have added this in today's release.  It will need some testing though to see if it works for all combinations of enabled/disabled systems.
